Doc/whatsnew/3.14.rst

```diff
@@ -135,6 +135,12 @@ symtable
 
   (Contributed by Bénédikt Tran in :gh:`120029`.)
 
+socket
+------
+
+Add constants for IsoTp CAN protocol.
```

Comment:
This might be silly, but is it IsoTP or ISO-TP? (if it's the latter, change also the NEWS entry).

Reply:
done